### ISSUE TITLE
evaluation metrics for pascal voc

### ISSUE THREAD

**Issue by bsvineethiitg** [xingyizhou/CenterNet#100]
Hi. I had a question about the VOC evaluations you included in the Appendix.

You seem to use the test2007 as validation set while training. And, also use the same test2007 set as "test" set to evaluate the results.

1. Can you please describe how are the test2007 sets in validation different from testing phase? Do you use flip data augmentation in one but not in other, etc? 

2. Are the results included in the Table for VOC Test set with the --flip_test tag or not (when running test.py)?

3. I have been finding that model_last.pth is performing better when test.py is run than model_best.pth when I train using your training scripts. If the validation set is same as test set, then shouldn't model_best.pth perform the best on test?

4. Can you verify if the metrics I am getting for your pretrained model for VOC-ctdet-384 match with your results? I am getting a mAP 72.55 where as in the paper 72.6 is reported. 

> with flip_test tag:
AP for aeroplane = 0.7331
AP for bicycle = 0.8123
AP for bird = 0.7127
AP for boat = 0.6261
AP for bottle = 0.4293
AP for bus = 0.7803
AP for car = 0.7869
AP for cat = 0.8639
AP for chair = 0.5602
AP for cow = 0.7822
AP for diningtable = 0.7282
AP for dog = 0.8057
AP for horse = 0.8369
AP for motorbike = 0.8186
AP for person = 0.7579
AP for pottedplant = 0.4770
AP for sheep = 0.7359
AP for sofa = 0.7418
AP for train = 0.8235
AP for tvmonitor = 0.6981
Mean AP = 0.7255

5. I am getting worse mAP when flip_test option is not used. Can you please clear my confusion around this tag? Which one should I choose for evaluating? 

Thanks a lot! Excellent work :)

**Comment by xingyizhou** [comment]
1. In testing we use flip but in validation we do not.
2. Yes.
3. Right, model_best is the model with the lowest loss. It is not necessarily the best in mAP.
4. Because we round 72.55 to 72.6.
5.  Of course you got worse mAP without flip test augmentation.
You can find the answer of all the above questions in the paper or code.

**Comment by pingnuttakrid** [comment]
how can i get mAP result ? when run test.py
now i can see only AP when run it .
